drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fw/host_rpu_sys_if.h

```diff
@@ -780,12 +780,25 @@ enum op_band {
 	BAND_24G
 };
 
+#define TWT_EXTEND_SP_EDCA  0x1
```

Comment:
```suggestion
#define TWT_TX_BEYOND_SP 0x1
```

No need to put the actual mechanism here.